src/setfit/modeling.py

```diff
@@ -196,20 +203,32 @@ def forward(
 
         return logits, probs
 
-    def predict_proba(self, x_test: torch.Tensor) -> torch.Tensor:
+    def predict_proba(self, x_test: Union[torch.Tensor, "ndarray"]) -> Union[torch.Tensor, "ndarray"]:
+        is_tensor = isinstance(x_test, torch.Tensor)  # Otherwise assume it's ndarray
+        if not is_tensor:
+            x_test = torch.Tensor(x_test).to(self.device)
         self.eval()
 
-        return self(x_test)[1]
+        out = self(x_test)[1]
+        if not is_tensor:
+            return out.detach().cpu().numpy()
+        return out
```

Comment:
We deprecated `np.ndarray` support for `SetFitHead` recently. We now rely on `SetFitModel` to always supply the Head with torch Tensors.